assets/scss/hero.scss

```diff
@@ -29,61 +53,85 @@
   }
 
   &.browse-jumbotron {
-    .container {
-      padding-bottom: 3rem;
+    margin-top: 2rem;
+    border-radius: 0.5rem;
+    overflow: hidden;
```

Comment:
Is the overflow hidden neccassary? What is overflowing?

Reply:
I've removed this